sharing/src/main/scala/io/delta/sharing/spark/DeltaSharingCDFUtils.scala

```diff
@@ -0,0 +1,105 @@
+/*
+ * Copyright (2021) The Delta Lake Project Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.delta.sharing.spark
+
+import java.lang.ref.WeakReference
+import java.nio.charset.StandardCharsets.UTF_8
+
+import org.apache.spark.sql.delta.catalog.DeltaTableV2
+import com.google.common.hash.Hashing
+import io.delta.sharing.client.DeltaSharingClient
+import io.delta.sharing.client.model.{Table => DeltaSharingTable}
+import org.apache.hadoop.fs.Path
+
+import org.apache.spark.delta.sharing.CachedTableManager
+import org.apache.spark.internal.Logging
+import org.apache.spark.sql.SQLContext
+import org.apache.spark.sql.sources.BaseRelation
+
+object DeltaSharingCDFUtils extends Logging {
+
+  private def getDuration(start: Long): Double = {
+    (System.currentTimeMillis() - start) / 1000.0
+  }
+
+  private[sharing] def prepareCDCRelation(
```

Comment:
nit: use CDF to be consistent with the file name

Reply:
updated.